SConstruct

```diff
@@ -164,12 +164,13 @@ env.Default(sky)
 # The testing infrastructure ignores "mode" specification (i.e. we only test optimized output).
 # (If we add support for code coverage output, this will likely need to change.)
 testBuildDirectory = pathjoin("tests", env["BUILDDIR"])
```

Comment:
note that you're still building the test object files and binary into `tests/build/*.o` and `tests/endless-sky-tests`, respectively

Reply:
Should be fixed with the the last updates I did.